lib/torque/leaflet/canvas_layer.js

```diff
@@ -60,11 +60,11 @@ L.CanvasLayer = L.Class.extend({
     return canvas;
   },
 
-  onAdd: function (map) {
-    this._map = map;
+  _layerAdd: function (e) {
```

Comment:
Yes, I tried it before... but It was failing, IDK why. But now I've tried again and it works... maybe some dep fail.

I'll leave it how it was.